app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -1,3 +1,10 @@
+h3
+  = title
+  - if @assessment.submission_downloadable? && course_users.map { |u| submissions[u] }.any? { |s| s && s.workflow_state != 'attempting' }
```

Comment:
I would prefer to keep it here as the `submissions` object here is a hash passed in by the parent view

https://github.com/Coursemology/coursemology2/blob/master/app/views/course/assessment/submission/submissions/index.html.slim#L8